packages/as-transformer/src/transformer.ts

```diff
@@ -117,6 +117,14 @@ export class Transformer extends TransformVisitor {
         Array.from(neededImports.keys()).forEach(
           (i) => (content = i + '\n' + content),
         );
+        const depsFilter = Array.from(
+          Array.from(neededImports.keys()).map((elem) => {
+            let splitted = elem.split('from ".');
+            if (splitted.length > 1 && splitted[1])
```

Comment:
can you change your test on splitted[1] to something more explicite (not empty I guesss). Autocast is a nightmare and this part of the code is sufficiently complex.